games-puzzle/color-lines/color_lines-0.6.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="A classic lines puzzle game"
+DESCRIPTION="This strategy Game will make you think and concentrate \
+for hours if you accept the challenge. The game will randomly place three \
+color balls on the game board at every round, you have to strategically \
+select one ball (click on ball) and move it to the most convenient location \
+(click on target square)."
+HOMEPAGE="https://code.google.com/archive/p/color-lines"
+COPYRIGHT="2011 Peter Kosyh"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/color-lines/lines_$portVersion.tar.gz"
+CHECKSUM_SHA256="e20c87dfcc8f940c8af7784dad12e66fa1cd10d0d97766979fef00eb17c0b0b3"
+SOURCE_DIR="lines-$portVersion"
+PATCHES="color_lines-$portVersion.patch"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	color_lines$secondaryArchSuffix = $portVersion
+	app:color_lines = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:libsdl_image$secondaryArchSuffix
+	lib:libsdl_mixer$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl$secondaryArchSuffix
+	devel:libsdl_image$secondaryArchSuffix
+	devel:libsdl_mixer$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+    make PREFIX=$prefix BINDIR=$appsDir GAMEDATADIR=$dataDir/color-lines/
```

Comment:
Almost perfect!
You just forgot to use real tabs for the indentation inside BUILD. (Same remark for INSTALL.)